CHANGELOG.md

```diff
@@ -2,7 +2,68 @@
 
 ## New Features:
 
-No changes to highlight.
+### Theme Sharing 🎨 🤝
+
+You can now share your gradio themes with the world!
+
+After creating a theme, you can upload it to the HuggingFace Hub to let others view it, use it, and build off of it!
+
+### Uploading
+There are two ways to upload a theme, via the theme class instance or the command line.
+
+1. Via the class instance
+```python
+my_theme.to_hub(repo_name="my_theme",
+                version="0.2.0",
+                hf_token="...")
+```
+
+2. Via the command line
+
+First save the theme to disk
+```python
+my_theme.dump(path="./", filename="my_theme.json")
```

Comment:
Is there a reason you separate `path` and `filename` instead of just accepting a filepath e.g. like `json.dump`?

Reply:
None 🙈 Making the change + pushing up now.